docs/runbook.md

```diff
@@ -41,8 +41,88 @@
 
 ### Configure Azure deployment
 
+Here is a sample yaml file for running testing on Azure platform. We need 1 Azure VM with at least 2 core count for testing.
+
+sample_azure.yml
+
+```yaml
+name: azure default
+extension:
+  paths:
+    - ../lisa/testsuites/basic
+variable:
+  - name: location
+    value: westus2
+  - name: reserve_environment
+    value: "no"
+notifier:
+  - type: html
+environment:
+  warn_as_error: false
+  allow_create: false
+  environments:
+    - nodes:
+        - type: requirement
+          core_count:
+            min: 2
+          azure:
```

Comment:
We should support the node specification in platform level. But for now, it should explain why environment must be provided for tests.